pkg/serverless/trace/inferredspan/inferred_span.go

```diff
@@ -72,3 +83,66 @@ func FilterFunctionTags(input map[string]string) map[string]string {
 
 	return output
 }
+
+func CreateInferredSpan(event string, inferredSpan InferredSpan) {
```

Comment:
The name of this function is a bit confusing. It is called `CreateInferredSpan` but you need to pass in an inferred span that was already created as an argument.